spec/experiments/string_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe String do
+  describe "#<<" do
```

Comment:
文字列なのでシングルクォーテーションにしましょう
```suggestion
  describe '#<<' do
```